### DISCUSSION TITLE
Voting for a new GrandOrgue icon, round 2

### DISCUSSION THREAD

**Discussion by oleg68** [GrandOrgue/grandorgue#965] [General]
After the first round of voting in #903, several icons have 4 votes and more. They participate in the second round of voting here.

Please vote with https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png. You can vote for several variants

Please discuss the icons at #734, not here.

**Comment by oleg68** [comment]
# 0
![0-32](https://user-images.githubusercontent.com/11586506/149657102-80fe116b-ce8e-427b-8173-052b0926e5b4.png)
![0-1024](https://user-images.githubusercontent.com/11586506/149657107-7f2efe79-4ac0-474f-8ac3-d930efe75d1b.png)

**Comment by oleg68** [comment]
# 17
![17-32](https://user-images.githubusercontent.com/11586506/149657143-f6d3e5fa-52e2-4e5d-b11c-933274fab448.png)
![17-1024](https://user-images.githubusercontent.com/11586506/149657147-00c69153-03ea-409b-8932-6a2a1d622f0b.png)

**Comment by oleg68** [comment]
# 23
![23-32](https://user-images.githubusercontent.com/11586506/149657175-d80337aa-b458-4364-95c2-74da109a4a44.png)
![23-1024](https://user-images.githubusercontent.com/11586506/149657177-aa77c3b7-d346-4490-b5bc-ff29110abc62.png)

**Comment by oleg68** [comment]
# 25
![25-32](https://user-images.githubusercontent.com/11586506/149657197-15cd22a9-375a-4296-b818-894a7482d67d.png)
![25-1024](https://user-images.githubusercontent.com/11586506/149657198-aca3c11b-a81a-4bf9-84b1-18be059db009.png)

**Comment by oleg68** [comment]
# 30
![30-32](https://user-images.githubusercontent.com/11586506/149657210-e00c3e6d-88bf-4fe9-bd7b-40de84ab0d73.png)
![30-1024](https://user-images.githubusercontent.com/11586506/149657213-874fa53c-b84f-4a52-8f28-8854313fdc68.png)

**Comment by MartinLAL** [comment]
![LALLIER_Martin_GrandOrgue_logo](https://user-images.githubusercontent.com/97849588/149673428-cd60f3ca-2880-482b-b754-d98141c87dd4.png)
